demoapp/build.gradle

```
@@ -44,6 +44,8 @@ android {
             aaptOptions {
                 cruncherEnabled false
             }
+
+            resValue("string", "andesui_demoapp_app_name", "Andes DEBUG")
```

```suggestion
            resValue("string", "andesui_demoapp_app_name", "Andes DEBUG v${libraryVersion}")
```
could help when someone is sharing an screenshot